_posts/2016-03-02-Lazily-Automation.markdown

```
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Being a Better Programmer When You're Actually Lazy."
+date: 2016-03-02 12:09
+author: orta
+categories: [automation, culture, teams]
+---
+
+I juggle a bunch of projects, both in Artsy and in the Open Source community. If you don't work with me directly, you'd be mistaken for believing that I was an organized person. I'm pretty far from that, and [don't really](https://github.com/artsy/mobile/issues/68) plan on changing that.
+
+However, I work with other people and very strongly believe the programming is mostly a social problem once you're good enough at writing code. It'd be hypocritical of me to not improve the people process side, so I try to automate something that makes me a better team-mate.
+
+I'm going to cover four things I've worked on lately to improve this: [Danger](https://github.com/danger/danger/), [GitHub-Clippers](https://github.com/orta/github-clippers) and improving how I write commits and prefixing my name on branches.
+
+<!-- more -->
+
+### Danger
+
+[Danger](https://github.com/danger/danger/) is a tool I co-created with a friend ([Felix Krause](https://github.com/krausefx/)), and [my girlfriend](https://github.com/dangermcshane). It came out of frustration that we couldn't easily hold ourselves accountable to better team processes. Requiring a code-reviewer to remember also remember things like "Add a CHANGELOG Entry" isn't much to ask, but it is "Yet Another Thing To Remember" for both the submitter and reviewer.
+
+It's turning into a really important part of our code-review, and the ideas it generates once we had it in place make it fun to add new rules. It means we can feel fast, and not have to think about process so much. Danger will tell us if we've missed something.
```

"feel fast" -> "fail fast"